Improbable/Stdlib/Improbable.Stdlib/Platform/Platform.cs

```diff
@@ -0,0 +1,236 @@
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading;
+using Google.Api.Gax.Grpc;
+using Google.Protobuf.Collections;
+using Google.Protobuf.WellKnownTypes;
+using Improbable.SpatialOS.Deployment.V1Alpha1;
+using Improbable.SpatialOS.Platform.Common;
+using Improbable.SpatialOS.PlayerAuth.V2Alpha1;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Improbable.Stdlib.Platform
+{
+    public static class Platform
+    {
+        public static string ToolbeltConfigDir =>
+            Environment.GetEnvironmentVariable("IMPROBABLE_CONFIG_DIR") ?? (RuntimeInformation.IsOSPlatform(OSPlatform.Windows)
+                ? Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.LocalApplicationData), ".improbable")
+                : Path.Combine(Environment.GetEnvironmentVariable("HOME"), ".improbable"));
+
+        private static PlatformCredential GetCredentials()
+        {
+            var refreshToken = File.ReadAllText(Path.Combine(ToolbeltConfigDir, "oauth2/oauth2_refresh_token"));
+            return new PlatformRefreshTokenCredential(refreshToken);
+        }
+
+        private static PlatformApiChannel GetLocalApiChannel()
+        {
+            return new PlatformApiChannel(GetCredentials(), new PlatformApiEndpoint("localhost", 9876, true));
+        }
+
+        public static Deployment StartLocal(in StartDeploymentOptions startDeployment)
```

Comment:
Any reason you aren't using async-await here? There should be async functions for the Platform SDK calls